universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -40,6 +44,7 @@
   private QuestionDefinition questionDefinition;
   private ProgramDefinition programDefinition;
   private UserRepository userRepository;
+  private UatProfile profile;
```

Comment:
(nit) should this be named more specifically so it is clear that it's a profile with a TI account?

Reply:
Done.